internal/support/notifications/rest_notification.go

```diff
@@ -218,17 +341,11 @@ func notificationBySenderHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	n, err := dbClient.GetNotificationBySender(vars["sender"], limitNum)
+	n, err := getNotificationsBySender(vars["sender"], limitNum)
 	if err != nil {
-		if err == db.ErrNotFound {
```

Comment:
Why did you remove this `ErrNotFound` check?

Reply:
This function does not return a ErrNotFound, it returns an empty json array when it doesn't find anything